run_tests.py

```diff
@@ -465,9 +467,11 @@ def run_test(testname, host, target):
                 ispc_cmd += " -O2"
 
         exe_wd = "."
-        if target.arch == "wasm32":
+        if target.arch == "wasm32" or target.arch == "wasm64":
             cc_cmd += " -D__WASM__"
-            options.wrapexe = "v8 --experimental-wasm-simd"
+            options.wrapexe = os.environ["EMSDK_NODE"]
```

Comment:
emsdk includes node.js. we can use it for wasm tests. it is easier to maintain and more practical than using v8 directly.
also, simd is standardized in recent versions and "--experimental-wasm-simd" option is no longer exists.